procedures.sql

```diff
@@ -0,0 +1,1136 @@
+CREATE PROCEDURE AddStudent
+    @first_name VARCHAR(50),
+    @last_name VARCHAR(50),
+    @address VARCHAR(200),
+    @email VARCHAR(50),
+    @phone VARCHAR(20)
+AS
+BEGIN
+    INSERT INTO Student (name, surname, address, email, phone_number)
+    VALUES (@first_name, @last_name, @address, @email, @phone);
+END;
+GO
+
+CREATE PROCEDURE AddTeacher
+    @first_name VARCHAR(50),
+    @last_name VARCHAR(50),
+    @address VARCHAR(200),
+    @email VARCHAR(50),
+    @phone VARCHAR(20)
+AS
+BEGIN
+    INSERT INTO Teacher (name, surname, address, email, phone_number)
+    VALUES (@first_name, @last_name, @address, @email, @phone);
+END;
+GO
+
+CREATE PROCEDURE AddTranslator
+    @first_name VARCHAR(50),
+    @last_name VARCHAR(50),
+    @address VARCHAR(200),
+    @email VARCHAR(50),
+    @phone VARCHAR(20),
+    @language VARCHAR(50)
+AS
+BEGIN
+    INSERT INTO Translator (name, surname, address, email, phone_number, language)
+    VALUES (@first_name, @last_name, @address, @email, @phone, @language);
+END;
+GO
+-- EXEC AddStudent 'Bartłomiej', 'Kozak', 'Kraków 607 Kapitol', 'b_kozak@student.agh.edu.pl', '123321123'
```

Comment:
leak of the private data. you'll be burnt in hell

Reply:
definitely, your first and last name in your project

Reply:
and email, and address :)